src/Akka.Persistence.Hosting/AkkaPersistenceHostingExtensions.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Akka.Configuration;
+using Akka.Hosting;
+using Akka.Persistence.Journal;
+using Akka.Util;
+
+namespace Akka.Persistence.Hosting
+{
+    public enum PersistenceMode
```

Comment:
Breaking change - deleted the two library-specific versions of this enum and consolidated them into a common enum. This will break compat, possibly, with Akka.Persistence.SqlServer and Akka.Persistence.PostgreSql.